### DISCUSSION TITLE
LSe inspired simulator

### DISCUSSION THREAD

**Discussion by R3dst0ne** [logisim-evolution/logisim-evolution#1868] [General]
I just stumbled over [Quantr Logic](https://www.quantr.foundation/project/?project=Quantr%20Logic) which shares a lot of similarities with LSe (at least the UI).
They are even comparing it (the UI) themselves in this picture [Lse vs Quantr Logic](https://www.quantr.foundation/wp-content/uploads/2023/07/Screenshot-2023-07-01-at-4.32.30-PM.png).
![Lse vs Quantr Logic](https://user-content.gitlab-static.net/e8807ac3f7881cac6ee3ef39c154b9491ec4a30a/68747470733a2f2f7777772e7175616e74722e666f756e646174696f6e2f77702d636f6e74656e742f75706c6f6164732f323032332f30372f53637265656e73686f742d323032332d30372d30312d61742d342e33322e33302d504d2e706e67)
After seeing the purple logo and reading the group name a couple of times, I remembered @quantrpeter, he asked a view questions here (#1354, #1616, #1705).

I had no time looking deeper into the code, but at first glance it does not look like they used code from LSe. This could have been a problem because of the [incompatibility of GLP and Apache V2](https://www.apache.org/licenses/GPL-compatibility.html) licenses.
Even if no LS(e)  code was used, I think it would be nice to state what they were inspired by.

While clicking through the source, I did not expect to find pornographic content. Looking at the [commit that introduced it](https://gitlab.com/quantr/eda/quantr-logic/-/commit/9a00d1659fb1daf8b9a0d8ccb1e0fe0d21537377) only raised more questions.

All in all, I'm not sure what to make of it.

**Comment by quantrpeter** [comment]
hello, we build from the ground up and our source is in gitlab https://gitlab.com/quantr/eda/quantr-logic

I was trying to modify LSE but failed, so we build a new one.
